Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -19,6 +19,8 @@ private func dispatchOnMainThread(action: KlaviyoAction) {
 
 // MARK: Objective-C
 
+@available(
+    iOS, deprecated: 9999, message: "Use either KlaviyoSDK for swift bindings or KlaviyoObjc for Objective-C bindings.")
```

Comment:
if you use the `deprecated` decorator without specifying a version number does it not deprecate for all versions?

Reply:
Also would it make sense to use `renamed` as that seems to let folks use the fix it option in xcode vs. having to manually find the method and replace it.

<img width="882" alt="image" src="https://user-images.githubusercontent.com/118314354/226743639-a8772c73-e447-4576-93fc-e8bd1ab8e703.png">